src/collections/_documentation/workflow/search.md

```diff
@@ -18,9 +28,41 @@ In the above there are four tokens:
 -   `server:web-8`
 -   `example error`
 
-The first two are standard search tokens, both using reserved keywords. The third is pointing to a custom tag sent by the SDK. The fourth is passed as part of the issue search query (which uses a CONTAINS match).
+The first two are standard search tokens, both using reserved keywords. The third is pointing to a custom tag sent by the SDK. The fourth is passed as part of the issue search query (which uses a `CONTAINS` match).
+
+### Advanced
+
+**Exclusion**
+
+By default, search terms are `AND`-ed together; they return the intersection of issues/events that match all search terms.
+
+To change this, you can use the negation operator `!` to _exclude_ a search parameter.
+
+```
+is:unresolved !user.email:example@customer.com
+```
+
+In the example above, the search query returns all Issues that are unresolved _and_ have not affected the user with the email address `example@customer.com`.
+
+**Wildcards**
+
+Search supports the wildcard operator `*` for values.
 
-The following tokens are reserved and known to Sentry:
+```
+browser:"Safari 11*"
+```
+
+In the example above, the search query will match on `browser` values like `"Safari 11.0.2"`, `"Safari 11.0.3"`, etc.
+
+## Search Properties
+
+In the examples above, we've highlighted a couple example properites you can search on: `is`, `user`, `server`, `browser`, etc. Below is a canonical list of all available search terms.
+
+### Issue Properties
+
+Issues are an aggregate of one or more events. Searchable properties include workflow status, assignment, aggregate counts, and age.
+
+Below is a list of Issue-level tokens reserved and known to Sentry:
```

Comment:
I totally get this sentence, but would a human being use it?

Reply:
I think it's okay. :) The line above "Issues are..." helps set the context of "Below is a list..."